apps/mobile/src/app/(tabs)/activity.tsx

```diff
@@ -16,7 +15,7 @@ export default function ActivityScreen() {
 
   return (
     <Screen>
-      <Screen.Header centerElement={<Text variant="label01">{pageTitle}</Text>} />
+      <Screen.Header leftElement={null} />
```

Comment:
So you can't go back here, only change tab back to `Home`? 

Maybe we can get rid of the `ActivityWidget` from the `Home` page then?

On the `Account` page accessing `Activity` will be a bit different than on `Home` as it will use the original way but that's OK